pkg/webhook/operatingsystemconfig/scripts/configure-containerd-registries.sh

```diff
@@ -28,10 +29,11 @@ function with_backoff {
     echo "[$run_id] Request failed! Retrying in $interval seconds..."
     sleep "$interval"
 
-    attempt=$(( attempt + 1 ))
-    if (( interval * 2 <= max_interval )); then
+    if (( attempt * (interval + 2) >= linear_duration && interval * 2 <= max_interval )); then
```

Comment:
I guess it would be easier to follow and maintain if we use `max_linear_attempts` instead of `linear_duration`.
The check in that case would be:
```suggestion
    if (( attempt > max_linear_attempts && interval * 2 <= max_interval )); then
```

We could also rename `max_attempts` to `max_total_attempts`. These `linear_duration=150` should be equal to `max_linear_attempts=30`, right?